subsys/bluetooth/controller/ll_sw/ull_sched.c

```diff
@@ -77,7 +78,8 @@ void ull_sched_after_mstr_slot_get(uint8_t user_id, uint32_t ticks_slot_abs,
 			}
 		}
 
-		LL_ASSERT(ret_cb == TICKER_STATUS_SUCCESS);
+		success = ret_cb == TICKER_STATUS_SUCCESS;
```

Comment:
```suggestion
		success = (ret_cb == TICKER_STATUS_SUCCESS);
```

Easier on eyes to differentiate the single `=` vs the double `==`.

Reply:
Done.